jenkins/bootstrap.py

```diff
@@ -967,6 +975,23 @@ def parse_args(arguments=None):
         '--clean',
         action='store_true',
         help='Clean the git repo before running tests.')
+    # Use "_no_-magic-env" because the default (current functionality)
+    # is to have magic environment, so the default value for this flag
+    # (false) needs to match this behavior.
+    parser.add_argument(
+        '--raw-env',
+        action='store_true',
+        help='Disable Jenkins magic environment')
+    # If specified, this is the (relative) path to jobs configuration directory
+    # the cloned repo.
+    parser.add_argument(
+        '--jobs-dir',
+        # The default behavior assumes that bootstrap.py is run from the jenkins/
+        # directory of the kubernetes/test-infra repo, and uses this assumption to
+        # find the path of the kubernetes/test-infra repo (assuming it has been
+        # cloned). It then uses the root of the kubernetes/test-infra repo as the
+        # implicit jobs-config-dir.
+        help='Explicit location for where to look for job scripts relative to repo root.')
```

Comment:
how about: "Look for job configuration inside this dir rather than relative to the script"?